tests/x11regressions/firefox/firefox_java.pm

```diff
@@ -16,45 +16,40 @@ use base "x11regressiontest";
 use testapi;
 
 sub java_testing {
+    my ($self) = @_;
+
     send_key "ctrl-t";
     send_key "alt-d";
     type_string "http://www.java.com/en/download/installed.jsp?detect=jre\n";
 
     wait_still_screen 3;
+    assert_and_click('firefox-java-agree-and-proceed') if check_screen('oracle-cookies-handling'), 0;
```

Comment:
Are you sure it should be `check_screen(…), 0` and not `check_screen(…, 0)`

Reply:
True, don't even know how this happened :) Will fix now.